src/Notifier.ts

```diff
@@ -386,7 +387,16 @@ export const Notifier = {
     },
 
     _evaluateEvent: function(ev: MatrixEvent) {
-        const room = MatrixClientPeg.get().getRoom(ev.getRoomId());
+        let roomId = ev.getRoomId();
+        if (CallHandler.instance.getSupportsVirtualRooms()) {
+            // Attempt to translate a virtual room to a native one
+            const nativeRoomId = window.mxVoipUserMapper.nativeRoomForVirtualRoom(roomId);
```

Comment:
Is there a way to pass that down to the component or have an import? Rather than relying on an indirect reference?

Reply:
ah, yes - it should be using sharedInstance()